pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```
@@ -17,6 +17,8 @@ lib.packagesFromDirectoryRecursive {
 
   elpaca = callPackage ./manual-packages/elpaca { inherit (pkgs) git; };
 
+  lua = callPackage ./manual-packages/lua/package.nix { inherit (pkgs) stdenv lua; };
```

```suggestion
  lua = callPackage ./manual-packages/lua { inherit (pkgs) lua; };
```

Please also move this line after lsp-bridge to keep them sorted.